### ISSUE TITLE
Is it possible to make this application not include a basic theme, but a classic one?

### ISSUE THREAD

**Issue by Slucket** [Ingan121/BasicThemer2#24]
Recently, I was trying to install the classic theme on Windows 10. I came across the Simple Classic Theme app, which works very mediocrely.  The taskbar is buggy, the context menu is displayed incorrectly, the stock task manager crashes.
As far as I know, Basic Themer 2 and SCT use the same method of changing the system style, only one application includes the classic theme, the other the basic one.
In BT2, I like the implementation of the exception system: you can specify which application will/won't use the basic theme. I also like that the taskbar stays normal, I don't have to use apps like RetroBar or something.
Question: is it possible to make this application not include a basic theme, but a classic one?

**Comment by yuvia08** [comment]
i dont see why you would want to use classic theme without the themed taskbar.. one workaround to win8 taskmgr crashing is to launch it before it launches classic theme?

**Comment by Anixx** [comment]
> As far as I know, Basic Themer 2 and SCT use the same method of changing the system style

No, absolutely. Applications like SCT or ClassicThemeTray simply close access to the memory area with the current theme whatever it is. Classic theme is not really a theme, but an absense of any.

**Comment by Anixx** [comment]
To address the OP's problem, I needed the same, so I use https://github.com/arukateru/basicthemer2-detours. I simply inluded Explorer.exe in the exclusion list:

![изображение](https://user-images.githubusercontent.com/1271736/226116494-42aecf18-bf3b-4a06-94cc-53135f5ce9a7.png)

You asked for exception system, it is already there.

**Comment by francesfarmer2** [comment]
Hi,

I was wondering why basicthemer2 doesn't display a basictheme with some themes but shrinks the buttons, this happens with aerolite (winaerotweaker) or all the sagorpibd XP themes and probably many others...

I customized  moonique's classic theme to look more like classic theme with windows 7 and shrinking buttons would be perfect ! because the only problem with my custom theme are the big buttons  ! there must be something that make this beahvior !??
